addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover_tests.js

```diff
@@ -199,6 +199,7 @@ QUnit.test('activity mark done popover mark done with feedback', async function
     let feedbackTextarea = document.querySelector('.o_ActivityMarkDonePopover_feedback');
     feedbackTextarea.focus();
     document.execCommand('insertText', false, 'This task is done');
+    feedbackTextarea.blur();
```

Comment:
No... This is no good. Making test implementation-dependent on that looks off. The blur looks out of place, and requiring that so that the field `feedback` is up-to-date is wrong... If you really need the feedback content in model, it should be seamless in test, so it should be auto-sync similarly to composer text input.

Also note that current implementation that saves feedback on blur and on click done (schedule next) is problematic, because this field is too deeply tight to flow-specific. In other words, feedback field is not declarative enough, so it's very prone to introduce bugs.